plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/s3select/S3SelectTestHelper.java

```diff
@@ -232,4 +242,46 @@ public void tearDown()
             heartbeatService = null;
         }
     }
+
+    int getTableSplitsCount(SchemaTableName table)
+    {
+        return getSplitsCount(
+                table,
+                getTransactionManager(),
+                getHiveConfig(),
+                getSplitManager());
+    }
+
+    MaterializedResult getFilteredTableResult(SchemaTableName table, ColumnHandle column)
+    {
+        try {
+            return filterTable(
+                    table,
+                    List.of(column),
+                    getTransactionManager(),
+                    getHiveConfig(),
+                    getPageSourceProvider(),
+                    getSplitManager());
+        }
+        catch (IOException ignored) {
```

Comment:
We should not reach this scenario and by returning null the test would fail anyway. Catch "ignored" approach is used in other tests as well across the project ([for example here](https://github.com/trinodb/trino/blob/fe608f2723842037ff620d612a706900e79c52c8/plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestReentrantBoundedExecutor.java#L47-L48)).